### PR TITLE
Re: Rita review on chapter 13

### PR DESCRIPTION
**Pull request by Xronophobe**
@ritaf-ORM for some reason I can't select your username for requesting review, so I'll just @ tag you.

### CODE REVIEW

**Review by hjwp — APPROVED** [review]

**Comment by Xronophobe** [comment]
@ritaf-ORM I'll wait for your approval as well before merging.

**Comment by ritaf-ORM** [comment]
@Xronophobe You don't have to wait for my approval before merging. But, thanks for addressing my comments!

**Pull request merged**
